html-css/week1/jamshedali/assignment-week-1/css/style.css

```
@@ -0,0 +1,173 @@
+body {
+  margin: auto;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+h1 {
+  text-align: center;
+}
+h2 {
+  margin: 0;
+  text-align: center;
+}
+h1 {
+  padding: 2%;
+}
+h2 {
+  color: #939393;
+}
+
+
+/* header */
+header {
+  width: 90%;
+  margin: auto;
+  color: white;
+  background-image: linear-gradient(to right, #fd7b46, #fe4e9b);
+}
+header h1 {
+  text-align: left;
+  padding-left: 4.6%;
+}
+.summary_total {
+  display: flex;
+  width: 90%;
+  margin: auto;
+  justify-content: space-between;
+  padding: 1.5% 0% 1.5% 0%;
+  border-bottom: 2px solid white;
+  font-size: 25px;
+}
+.grand_total {
+  display: flex;
+  width: 90%;
+  margin: auto;
+  justify-content: space-between;
+  padding: 1.5% 0% 1.5% 0%;
+  font-size: 25px;
+  font-weight: bold;
+}
+/* end header */
+
+
+/* main start */
+main {
+  width: 90%;
+  margin: auto;
+}
+main button {
+  width: 44%;
+  padding: 2%;
+  margin: 1%;
+  border-radius: 10px;
+  cursor: pointer;
+  box-shadow: 10px 10px 20px grey;
+  border: none;
+  background-color: white;
+}
+main button img {
+  max-width: 100%;
+  height: auto;
+  display: block;
+  margin: auto;
+}
+.active_button {
+  border: 5px solid #fe4e9a;
+}
+main p {
+  text-align: center;
+  margin: 1%;
+}
+
+
+/* main-> card information */
+.cardinfo_container {
+  width: 90%;
+  margin: auto;
+  display: flex;
+  gap: 3%;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+}
+.cardinfo_itemone {
+  flex-grow: 5;
+}
+.cardinfo_itemtwo {
+  flex-grow: 1;
+}
+.cardinfo_itemthree {
+  flex-grow: 2;
+}
+
+.cardinfo_container label {
+  font-size: 24px;
+  color: #939393;
+  display: block;
+  text-align: center;
+  padding: 15px;
+}
+.cardinfo_container input {
+  width: 100%;
+  box-sizing: border-box;
+  padding: 33px;
+  height: 50px;
+  border-radius: 2px;
+  border: 2px solid #adadad;
+  background-color: #f3f1f2;
+  font-size: 18px;
+  background-image: url("../images/question_mark.png");
+  background-repeat: no-repeat;
+  background-position: 98% 40%;
+}
+/* removing question (expiry field) */
+.expiry_data_filed {
+  background-image: none !important;
+}
+.cardinfo_container input::placeholder {
+  color: #939393;
+}
+/* main-> card information end */
+
+.buynow_button {
```

Rename to more content agnostic name that would allow you easily to reuse the same class elsewhere
```suggestion
.primary-button {
```

---

you are right make it primary for other pages or in other sections for reuse 

.primary-button {
  padding: 0;
  margin-top: 4%;
  border-radius: 100px;
  cursor: pointer;
  box-shadow: none;
  border: none;
  background-image: linear-gradient(to right, #fd7b46, #fe4e9b);
  display: block;
}
.primary-button h1 {
  color: white;
}
(done)